src/app/CommandHandler.cpp

```diff
@@ -125,6 +126,15 @@ CHIP_ERROR CommandHandler::ProcessInvokeRequest(System::PacketBufferHandle && pa
     }
 
     invokeRequests.GetReader(&invokeRequestsReader);
+
+    {
+        // We don't support handling multiple commands but the protocol is ready to support it in the future, reject all of them and
+        // IM Engine will send a status response.
+        size_t commandCount = 0;
+        TLV::Utilities::Count(invokeRequestsReader, commandCount, false /* recurse */);
+        VerifyOrReturnError(commandCount == 1, CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE);
```

Comment:
What is the observable behavior here on the client side?

Reply:
It will reject to encode such payload (CHIP_ERROR_INCORRECT_STATE)

Reply:
No, I mean if a client that is not the SDK sends multiple commands in a single invoke, what behavior will it observe?  Will it get back an error StatusResponse (if so what kind)?  Will it get back nothing?  Something else?

Reply:
This is what the TestCommandHandlerRejectMultipleCommands covers, it will get a StatusResponse.